stack.yaml

```diff
@@ -1,9 +1,10 @@
-resolver: lts-9.12
+resolver: lts-11.8
 packages:
 - '.'
 extra-deps:
-- haskell-src-exts-1.20.0
-- descriptive-0.9.4
+- haskell-src-exts-1.20.2
+- git: https://github.com/robertjlooby/descriptive
```

Comment:
Can flip this back once https://github.com/chrisdone/descriptive/pull/4 is released